packages/react-composites/src/composites/CallComposite/components/SidePane/usePeoplePane.tsx

```diff
@@ -183,6 +358,84 @@ export const usePeoplePane = (props: {
           disabled: isMuted
         });
       }
+
+      /* @conditional-compile-remove(media-access) */
+      const remoteParticipant = remoteParticipants?.[participantId];
+      /* @conditional-compile-remove(media-access) */
+      if (
+        remoteParticipant?.mediaAccess &&
+        !remoteParticipant.mediaAccess.isAudioPermitted &&
+        remoteParticipant?.role === 'Attendee' &&
+        onPermitAudio
+      ) {
+        _defaultMenuItems.push({
+          key: 'permit-audio',
+          text: localeStrings.permitAudioMenuLabel,
+          iconProps: {
+            iconName: 'ControlButtonMicOn',
+            styles: { root: { lineHeight: '1rem', textAlign: 'center' } }
+          },
+          onClick: () => {
+            onPermitAudio([participantId]);
+          },
+          'data-ui-id': 'participant-item-permit-microphone-button',
+          ariaLabel: localeStrings.permitAudioMenuLabel
+        });
+      }
+      /* @conditional-compile-remove(media-access) */
+      if (remoteParticipant?.mediaAccess?.isAudioPermitted && remoteParticipant?.role === 'Attendee' && onForbidAudio) {
+        _defaultMenuItems.push({
+          key: 'forbid-audio',
+          text: localeStrings.forbidAudioMenuLabel,
+          iconProps: {
+            iconName: 'ControlButtonMicProhibited',
+            styles: { root: { lineHeight: '1rem', textAlign: 'center' } }
+          },
+          onClick: () => {
+            onForbidAudio([participantId]);
+          },
+          'data-ui-id': 'participant-item-forbid-microphone-button',
+          ariaLabel: localeStrings.forbidAudioMenuLabel
+        });
+      }
+      /* @conditional-compile-remove(media-access) */
+      if (
+        remoteParticipant?.mediaAccess &&
+        !remoteParticipant.mediaAccess.isVideoPermitted &&
+        remoteParticipant?.role === 'Attendee' &&
+        onPermitVideo
+      ) {
+        _defaultMenuItems.push({
+          key: 'permit-video',
+          text: localeStrings.permitVideoMenuLabel,
+          iconProps: {
+            iconName: 'ControlButtonCameraOn',
+            styles: { root: { lineHeight: '1rem', textAlign: 'center' } }
+          },
+          onClick: () => {
+            onPermitVideo([participantId]);
+          },
+          'data-ui-id': 'participant-item-permit-camera-button',
+          ariaLabel: localeStrings.permitVideoMenuLabel
+        });
+      }
+      /* @conditional-compile-remove(media-access) */
+      if (remoteParticipant?.mediaAccess?.isVideoPermitted && remoteParticipant?.role === 'Attendee' && onForbidVideo) {
+        _defaultMenuItems.push({
+          key: 'forbid-video',
+          text: localeStrings.forbidVideoMenuLabel,
+          iconProps: {
+            iconName: 'ControlButtonCameraProhibitedSmall',
+            styles: { root: { lineHeight: '1rem', textAlign: 'center' } }
+          },
+          onClick: () => {
+            onForbidVideo([participantId]);
```

Comment:
Since the composite API can be flat for function on the adapter it might be simpler to create a helper in this hook to make a toggle from the adapter functions to cut down on the lines of code to maintain. 

```ts
const toggleForbidParticipantVideo = useCallback((participant) => {
if (participant.mediaAccess?.isVideoPermitted){
    onForbidVideo([participant.userId]);
} else {
    onPermitVideo([participant.userId])
}
},[onForbidVideo, onPermitVideo]);
```
Or something like this, then use this here. I think this might allow us to half the number of menu options being pushed about here

Reply:
This toggle already discussed in api review meeting, reviewers want separate function forbid and permit.